méli-mélo/2023-09-distance-calculator/index.html

```diff
@@ -0,0 +1,309 @@
+	"dateModified": "2023-09-08",
+	"script": [
+		"js/gc-distance-calculator.js"
+	]
+}
+---
+<p>Using the <a href="https://natural-resources.canada.ca/science-and-data/science-and-research/earth-sciences/geography/topographic-information/web-services/geolocation-service/17304">Natural Resources Canada Geogratis Geolocation service</a> to provide the distance between a known set of coordinates and an address provided by the user.</p>
+
+<h2>GCWeb implementation plan</h2>
+
+{% assign implPlan = site.pages | where: "output", "false" | where: "componentName", "2023-09-distance-calculator" | first %}
+
+<ul>
+{% for deliverable in implPlan.implementationPlan %}
+	<li>{{ deliverable.due }} - {{ deliverable.what }}</li>
+{% endfor %}
+</ul>
+
+<p>Todo and for future consideration</p>
+<ul>
+{% for todo in implPlan.todos %}
+	<li>{{ todo }}</li>
+{% endfor %}
+</ul>
+
+<p>Sponsor: {{ implPlan.sponsor }}</p>
+
+<h2>Classes used</h2>
+<ul class="colcount-sm-2 colcount-md-4">
+<li><code>gc-distance-calculator</code></li>
+</ul>
+
+
+<h2>Distance calculator settings</h2>
+
+<pre><code>
+"form":".gc-distance-form",
+"location":".gc-distance-location",
+"selector":".distance-item",
+"section":".distance-items",
+"target":".distance",
+"sort":".distance-sort",
+"display":{"selector":".distance-visible","removeClass":"hidden"},
+"name":".distance-title"
+</code></pre>
+
+<dl>
+<dt>form</dt>
+<dd>
+<code>"form":".gc-distance-form"</code>
+<p>Selector for form where location is submitted</p>
+</dd>
+<dt>location</dt>
+<dd>
+<code>"location":".gc-distance-location"</code>
+<p>Selector for input where location is entered</p>
+</dd>
+<dt>selector</dt>
+<dd>
+<code>"selector":".distance-item"</code>
+<p>Selector identifying the items in the collection to be updated with distance.</p>
+</dd>
+<dt>section</dt>
+<dd>
+<code>"section":".distance-items"</code>
+<p>Selector defining the collection of items</p>
+</dd>
+<dt>target</dt>
+<dd>
+<code>"target":".distance"</code>
+<p>Selector to identify element to update with formatted distance</p>
+</dd>
+<dt>sort</dt>
+<dd>
+<code>"sort":".distance-sort"</code>
+<p>Selector to identify element to update with unformatted distance number</p>
+</dd>
+<dt>display</dt>
+<dd>
+<code>"display":{"selector":".distance-visible","removeClass":"hidden"}</code>
+<p>Object with options to display content when distance is calculated.</p>
+	<dl>
+	<dt>selector</dt>
+	<dd>
+	<code>"selector":".distance-visible"</code>
+	<p>Selector for all elements to be revealed</p>
+	</dd>
+	<dt>removeClass</dt>
+	<dd>
+	<code>"removeClass":"hidden"</code>
+	<p>Class to be removed when results are displayed</p>
+	</dd>
+	</dl>
+</dd>
+<dt>name</dt>
+<dd>
+<code>"name":".distance-title"</code>
+<p>Selector to identify element to update with returned name for provided location</p>
+</dd>
+</dl>
+
+
+
+<section id="filteredEventsList" class="wb-tagfilter provisional" data-gc-distance='{
+		"form":".gc-distance-form",
+		"location":"gc-distance-location",
```

Comment:
I think this is a Class selector and not a web component (element) selector
```suggestion
		"location":".gc-distance-location",
```